pkg/api/types.go

```diff
@@ -1865,3 +1863,39 @@ type ComponentStatusList struct {
 
 	Items []ComponentStatus `json:"items"`
 }
+
+// SecurityContext holds security configuration that will be applied to a container.  SecurityContext
```

Comment:
Not sure I follow, the SecurityContext field is on the Container struct in the internal api and v1beta* apis.